src/setuptools.rs

```diff
@@ -13,7 +13,8 @@ static SETUPTOOLS_TEMPLATES: &[(&str, &str)] = &builtin_templates!["setuptools"
     ("md.README", "README.md"),
     ("py.__init__", "__init__.py"),
     ("py.__main__", "__main__.py"),
-    ("py.test", "test.py"),
+    ("py.test", "test-sum.py"),
```

Comment:
Why the rename?

Reply:
Because if we use the name `test.py`, `pytest` does not run any test. It is necessary add a name after the prefix `test`